.github/dependabot.yml

```diff
@@ -0,0 +1,73 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+
+version: 2
+updates:
+  # github actions
+  - package-ecosystem: "github-actions"
+    target-branch: "release-v1.12.0"
```

Comment:
For values that are the same for each directory (like target-branch), can we set it to a variable so it only needs to be updated in one spot?

Reply:
> For values that are the same for each directory (like target-branch), can we set it to a variable so it only needs to be updated in one spot?

No, dependabot does not support that and it reads the file directly

Reply:
Is it possible to automate the changing of the target-branch when a new release branch is created? Like another github action. Otherwise this is another thing we have to remember to maintain.

Reply:
Sure, a GitHub workflow is nothing more than automation that runs in the context of a repo.

Perhaps you need an action that gets run manually that creates your new release branch, updates the config values in this dependabot file, and issues a PR as necessary.